api/src/main/java/com/cloud/network/vpc/VpcProvisioningService.java

```diff
@@ -41,12 +43,9 @@ public VpcOffering createVpcOffering(String name, String displayText, List<Strin
     public boolean deleteVpcOffering(long offId);
 
     /**
-     * @param vpcOffId
-     * @param vpcOfferingName
-     * @param displayText
-     * @param state
+     * @param vpcOfferingCmd
      * @return
      */
-    public VpcOffering updateVpcOffering(long vpcOffId, String vpcOfferingName, String displayText, String state);
+    public VpcOffering updateVpcOffering(final UpdateVPCOfferingCmd vpcOfferingCmd);
```

Comment:
not sure if it is a biggy but,... any external plugins may suffer from this service api. better to add a new method and mark it deprecated

Reply:
Pretty sure there wasn't any internal plugin Daan, but I think it's a good practice in general. Doing the change now.

Reply:
Fixed @DaanHoogland . Please take a look.